audio-worklet/design-pattern/awn/Makefile

```diff
@@ -4,9 +4,9 @@ build: $(DEPS)
 	@emcc --bind -O1 -s WASM=1 \
 		-s BINARYEN_ASYNC_COMPILATION=0 -s SINGLE_FILE=1 \
 		AudioWorkletProcessorKernel.cc \
-		-o audio-worklet-module.js \
+		-o audio-worklet.wasmmodule.js \
```

Comment:
For the efficient pattern matching, all the WASM module code will be named like this.